ingestion/src/metadata/orm_profiler/processor/orm_profiler.py

```diff
@@ -50,16 +48,12 @@
 from metadata.ingestion.ometa.ometa_api import OpenMetadata
 from metadata.orm_profiler.api.models import ProfilerProcessorConfig, ProfilerResponse
 from metadata.orm_profiler.interfaces.interface_protocol import InterfaceProtocol
-from metadata.orm_profiler.metrics.registry import Metrics
-from metadata.orm_profiler.orm.converter import ometa_to_orm
-from metadata.orm_profiler.profiler.core import Profiler
-from metadata.orm_profiler.profiler.default import DefaultProfiler, get_default_metrics
 from metadata.orm_profiler.profiler.handle_partition import (
     get_partition_cols,
     is_partitioned,
 )
 from metadata.orm_profiler.validations.models import TestCase as CLIConfigTestCase
-from metadata.orm_profiler.validations.models import TestDef
+from metadata.orm_profiler.validations.models import TestCase as TestCaseConfig
 from metadata.utils.helpers import get_start_and_end
 
 logger = logging.getLogger("ORM Profiler Workflow")
```

Comment:
nit - I think we have some utils functions to pick up the right logger without passing a string